src/pybind/mgr/dashboard/controllers/host.py

```diff
@@ -1,12 +1,104 @@
 # -*- coding: utf-8 -*-
 from __future__ import absolute_import
+import copy
 
-from . import ApiController, RESTController
+from mgr_util import merge_dicts
+from . import ApiController, RESTController, Task
 from .. import mgr
+from ..exceptions import DashboardException
 from ..security import Scope
+from ..services.orchestrator import OrchClient
+from ..services.exception import handle_orchestrator_error
+
+
+def host_task(name, metadata, wait_for=10.0):
+    return Task("host/{}".format(name), metadata, wait_for)
+
+
+def merge_hosts_by_hostname(ceph_hosts, orch_hosts):
+    """Merge Ceph hosts with orchestrator hosts by hostnames.
```

Comment:
@jmolmo + @tserong In the past, I've seen uses thatused different host aliases for different network interfaces of physical hosts. At least for DeepSea, I'm expecting this to break in some corner cases. Are there similar uses cases with ceph-ansible?